### ISSUE TITLE
Intermittent kitchen test failures

### ISSUE THREAD

**Issue by flah00** [test-kitchen/kitchen-ec2#228]
This issue only appears from time to time and appears across cookbooks. It only happens on my ubuntu (14.04) jenkins (1.647) slave. I have never run into this issue on os x 10.10.

```
$ bundle list
Gems included by the bundle:
  * addressable (2.3.8)
  * aws-sdk (2.1.36)
  * aws-sdk-core (2.1.36)
  * aws-sdk-resources (2.1.36)
  * berkshelf (4.0.1)
  * berkshelf-api-client (2.0.0)
  * buff-config (1.0.1)
  * buff-extensions (1.0.0)
  * buff-ignore (1.1.1)
  * buff-ruby_engine (0.1.0)
  * buff-shell_out (0.2.0)
  * builder (3.2.2)
  * bundler (1.10.6)
  * celluloid (0.16.0)
  * celluloid-io (0.16.2)
  * chef (12.4.1)
  * chef-config (12.4.1)
  * chef-zero (4.3.0)
  * chefspec (4.3.0)
  * cleanroom (1.0.0)
  * coderay (1.1.0)
  * dep-selector-libgecode (1.0.2)
  * dep_selector (1.0.3)
  * diff-lcs (1.2.5)
  * erubis (2.7.0)
  * excon (0.45.4)
  * faraday (0.9.2)
  * fauxhai (2.3.0)
  * ffi (1.9.10)
  * ffi-yajl (2.2.2)
  * hashie (2.1.2)
  * highline (1.7.5)
  * hitimes (1.2.3)
  * httpclient (2.6.0.1)
  * ipaddress (0.8.0)
  * jmespath (1.1.3)
  * json (1.8.3)
  * kitchen-ec2 (0.10.0)
  * libyajl2 (1.2.0)
  * method_source (0.8.2)
  * mime-types (2.6.2)
  * minitar (0.5.4)
  * mixlib-authentication (1.3.0)
  * mixlib-cli (1.5.0)
  * mixlib-config (2.2.1)
  * mixlib-log (1.6.0)
  * mixlib-shellout (2.2.1)
  * multi_json (1.11.2)
  * multipart-post (2.0.0)
  * net-scp (1.2.1)
  * net-ssh (2.9.2)
  * net-ssh-gateway (1.2.0)
  * net-ssh-multi (1.2.1)
  * net-telnet (0.1.1)
  * nio4r (1.1.1)
  * octokit (3.8.0)
  * ohai (8.5.1)
  * plist (3.1.0)
  * pry (0.10.1)
  * rack (1.6.4)
  * rake (10.4.2)
  * retryable (2.0.2)
  * ridley (4.3.2)
  * rspec (3.3.0)
  * rspec-core (3.3.2)
  * rspec-expectations (3.3.1)
  * rspec-its (1.2.0)
  * rspec-mocks (3.3.2)
  * rspec-support (3.3.0)
  * rspec_junit_formatter (0.2.3)
  * safe_yaml (1.0.4)
  * sawyer (0.6.0)
  * semverse (1.2.1)
  * serverspec (2.23.1)
  * sfl (2.2)
  * slop (3.6.0)
  * solve (1.2.1)
  * specinfra (2.43.4)
  * syslog-logger (1.6.8)
  * systemu (2.6.5)
  * test-kitchen (1.4.2)
  * thor (0.19.1)
  * timers (4.0.4)
  * uuidtools (2.1.5)
  * varia_model (0.4.1)
  * wmi-lite (1.0.0)
```

```
$ berks list
Cookbooks installed by your Berksfile:
  * adaptly-system (0.5.26) from source at .
  * apt (2.7.0)
  * aws (1.0.0)
  * build-essential (2.1.3)
  * chef_handler (1.1.4)
  * datadog (2.2.0) from https://github.com/DataDog/chef-datadog.git (at master)
  * java (1.22.0)
  * logrotate (1.9.2)
  * monit (0.7.4) from https://github.com/apsoto/monit.git (at master)
  * ohai (2.0.1)
  * opsline-dogwatch (0.1.0) from https://github.com/opsline/chef-dogwatch.git (at master)
  * prep (1.0.0) from source at test/fixtures/cookbooks/prep
  * python (1.4.6)
  * rbenv (0.8.1) from https://github.com/chef-rbenv/ruby_rbenv.git (at v0.8.1)
  * redisio (2.3.0)
  * ruby_build (0.8.0)
  * sudo (2.7.2)
  * sumologic-collector (1.2.3) from https://github.com/SumoLogic/sumologic-collector-chef-cookbook (at master)
  * swap (0.3.8) from https://github.com/sethvargo-cookbooks/swap.git (at master)
  * ulimit (0.3.3)
  * windows (1.39.1)
  * yum (3.5.1)
  * yum-epel (0.6.2)
```

``` yaml

---
driver:
  shared_credentials_profile: prod-west
  aws_ssh_key_id: key2
  name: ec2
  associate_public_ip: true
  availability_zone: us-west-2b
  block_device_mappings:
    - ebs_device_name: /dev/sda1
      ebs_volume_size: 100
      ebs_volume_type: gp2
      ebs_optimized: false
      ebs_delete_on_termination: true
  instance_type: m3.medium
  security_group_ids: [sg-06e2ab63]
  region: us-west-2
  subnet_id: subnet-1f2a356b
  iam_profile_name: chef-testing

provisioner:
  name: chef_zero
  data_bags_path: test/integration/data_bags
  environments_path: test/integration/environments
  client_rb:
    environment: production

transport:
  ssh_key: ~/.chef/key2.pem

platforms:
  - name: ami-2.6
    driver:
      image_id: ami-a5ecfec4
    transport:
      username: ubuntu

suites:
  - name: monit_and_forwarding
    chef_client:
      config:
        log_level: ":debug"
    run_list:
      - recipe[prep::default]
      - recipe[prep::enable-forwarding]
      - recipe[adaptly-system::monit]
      - recipe[adaptly-system::disable-forwarding]
    attributes:

  - name: qurd
    chef_client:
      config:
        log_level: ":debug"
    run_list:
      - recipe[apt]
      - recipe[prep::default]
      - recipe[adaptly-system::qurd]
    attributes:
      adaptly_system:
        qurd:
          config: /etc/qurd/qurd.yml
          user: nobody
          group: nogroup

  - name: monitoring
    chef_client:
      config:
        log_level: ":debug"
    run_list:
      - recipe[apt]
      - recipe[prep::default]
      - recipe[adaptly-system::monitoring]
    attributes:
      adaptly_system:

  - name: swap_available
    driver:
      instance_type: r3.large
      image_id: ami-02ebf963
    chef_client:
      config:
        log_level: ":debug"
    run_list:
      - recipe[apt]
      - recipe[prep::default]
      - recipe[adaptly-system::swap]
    attributes:
      adaptly_system:
        swap:
          enabled: true

  - name: swap_unavailable
    driver:
      instance_type: t2.medium
      image_id: ami-02ebf963
    chef_client:
      config:
        log_level: ":debug"
    run_list:
      - recipe[apt]
      - recipe[prep::default]
      - recipe[adaptly-system::swap]
    attributes:
      adaptly_system:
        swap:
          enabled: true

  - name: swap_disabled
    chef_client:
      config:
        log_level: ":debug"
    run_list:
      - recipe[apt]
      - recipe[prep::default]
      - recipe[adaptly-system::swap]
    attributes:
      adaptly_system:
        swap:
          enabled: false

  - name: ebs
    driver:
      block_device_mappings:
        - ebs_device_name: /dev/xvdf
          ebs_volume_size: 10
          ebs_delete_on_termination: true
        - ebs_device_name: /dev/xvdg
          ebs_volume_size: 10
          ebs_delete_on_termination: true
    chef_client:
      config:
        log_level: ":debug"
    run_list:
      - recipe[apt]
      - recipe[prep::default]
      - recipe[adaptly-system::ebs]
    attributes:
      adaptly_system:
        ebs:
          - device: /dev/xvdf
            mount_point: /test
            user: ubuntu
            group: ubuntu
          - device: /dev/xvdg
            mount_point: /test/deep
            create_user: true
            user: ebsuser
            group: ebsgroup

  - name: ebs_reused
    driver:
      block_device_mappings:
        - ebs_device_name: /dev/xvdf
          ebs_volume_size: 10
          ebs_delete_on_termination: true
    chef_client:
      config:
        log_level: ":debug"
    run_list:
      - recipe[apt]
      - recipe[prep::default]
      - recipe[prep::ebs-data]
      - recipe[adaptly-system::ebs]
    attributes:
      adaptly_system:
        ebs:
          - device: /dev/xvdf
            mount_point: /test
            user: ubuntu
            group: ubuntu

  - name: oom
    chef_client:
      config:
        log_level: ":debug"
    run_list:
      - recipe[apt]
      - recipe[prep::default]
      - recipe[prep::oom-score-adj]
      - recipe[adaptly-system::oom-score-adj]
    attributes:
      adaptly_system:
        oom_score_adj:
          - pid_file: /run/atd.pid
            score: "-16"
          - pid_file: /run/rsyslogd.pid
          - pid_file: /run/sshd.pid
            score: -1000
          - pid_file: /tmp/test.pid
```

```
I, [2016-02-22T21:04:01.075533 #30561]  INFO -- Kitchen: -----> Starting Kitchen (v1.4.2)
I, [2016-02-22T21:04:01.362841 #30561]  INFO -- Kitchen: -----> Cleaning up any prior instances of <monit-and-forwarding-adaptly-ami-26>
I, [2016-02-22T21:04:01.362921 #30561]  INFO -- Kitchen: -----> Cleaning up any prior instances of <qurd-adaptly-ami-26>
I, [2016-02-22T21:04:01.362982 #30561]  INFO -- Kitchen: -----> Cleaning up any prior instances of <monitoring-adaptly-ami-26>
I, [2016-02-22T21:04:01.363589 #30561]  INFO -- Kitchen: -----> Destroying <monit-and-forwarding-adaptly-ami-26>...
I, [2016-02-22T21:04:01.363652 #30561]  INFO -- Kitchen: -----> Cleaning up any prior instances of <swap-available-adaptly-ami-26>
I, [2016-02-22T21:04:01.363805 #30561]  INFO -- Kitchen: -----> Cleaning up any prior instances of <swap-unavailable-adaptly-ami-26>
I, [2016-02-22T21:04:01.363861 #30561]  INFO -- Kitchen: -----> Cleaning up any prior instances of <swap-disabled-adaptly-ami-26>
I, [2016-02-22T21:04:01.365845 #30561]  INFO -- Kitchen: -----> Destroying <swap-disabled-adaptly-ami-26>...
I, [2016-02-22T21:04:01.366283 #30561]  INFO -- Kitchen: -----> Testing <swap-disabled-adaptly-ami-26>
I, [2016-02-22T21:04:01.366379 #30561]  INFO -- Kitchen: -----> Creating <swap-disabled-adaptly-ami-26>...
I, [2016-02-22T21:04:01.364278 #30561]  INFO -- Kitchen: -----> Destroying <qurd-adaptly-ami-26>...
I, [2016-02-22T21:04:01.365138 #30561]  INFO -- Kitchen: -----> Destroying <monitoring-adaptly-ami-26>...
I, [2016-02-22T21:04:01.365245 #30561]  INFO -- Kitchen: -----> Destroying <swap-available-adaptly-ami-26>...
I, [2016-02-22T21:04:01.365602 #30561]  INFO -- Kitchen: -----> Testing <monit-and-forwarding-adaptly-ami-26>
I, [2016-02-22T21:04:01.365714 #30561]  INFO -- Kitchen: -----> Destroying <swap-unavailable-adaptly-ami-26>...
I, [2016-02-22T21:04:01.364089 #30561]  INFO -- Kitchen: -----> Cleaning up any prior instances of <oom-adaptly-ami-26>
I, [2016-02-22T21:04:01.363912 #30561]  INFO -- Kitchen: -----> Cleaning up any prior instances of <ebs-adaptly-ami-26>
I, [2016-02-22T21:04:01.377014 #30561]  INFO -- Kitchen: -----> Destroying <ebs-adaptly-ami-26>...
I, [2016-02-22T21:04:01.364037 #30561]  INFO -- Kitchen: -----> Cleaning up any prior instances of <ebs-reused-adaptly-ami-26>
I, [2016-02-22T21:04:01.368159 #30561]  INFO -- Kitchen: -----> Testing <swap-available-adaptly-ami-26>
I, [2016-02-22T21:04:01.377763 #30561]  INFO -- Kitchen: -----> Creating <swap-available-adaptly-ami-26>...
I, [2016-02-22T21:04:01.368269 #30561]  INFO -- Kitchen: -----> Testing <qurd-adaptly-ami-26>
I, [2016-02-22T21:04:01.368978 #30561]  INFO -- Kitchen: -----> Testing <swap-unavailable-adaptly-ami-26>
I, [2016-02-22T21:04:01.378677 #30561]  INFO -- Kitchen: -----> Creating <swap-unavailable-adaptly-ami-26>...
I, [2016-02-22T21:04:01.367714 #30561]  INFO -- Kitchen: -----> Testing <monitoring-adaptly-ami-26>
I, [2016-02-22T21:04:01.377551 #30561]  INFO -- Kitchen: -----> Testing <ebs-adaptly-ami-26>
I, [2016-02-22T21:04:01.379828 #30561]  INFO -- Kitchen: -----> Creating <ebs-adaptly-ami-26>...
I, [2016-02-22T21:04:01.368379 #30561]  INFO -- Kitchen: -----> Creating <monit-and-forwarding-adaptly-ami-26>...
I, [2016-02-22T21:04:01.378512 #30561]  INFO -- Kitchen: -----> Destroying <ebs-reused-adaptly-ami-26>...
I, [2016-02-22T21:04:01.379625 #30561]  INFO -- Kitchen: -----> Creating <monitoring-adaptly-ami-26>...
I, [2016-02-22T21:04:01.376694 #30561]  INFO -- Kitchen: -----> Destroying <oom-adaptly-ami-26>...
I, [2016-02-22T21:04:01.378456 #30561]  INFO -- Kitchen: -----> Creating <qurd-adaptly-ami-26>...
I, [2016-02-22T21:04:01.381624 #30561]  INFO -- Kitchen: -----> Testing <ebs-reused-adaptly-ami-26>
I, [2016-02-22T21:04:01.383853 #30561]  INFO -- Kitchen: -----> Creating <ebs-reused-adaptly-ami-26>...
I, [2016-02-22T21:04:01.384836 #30561]  INFO -- Kitchen: -----> Testing <oom-adaptly-ami-26>
I, [2016-02-22T21:04:01.385042 #30561]  INFO -- Kitchen: -----> Creating <oom-adaptly-ami-26>...
I, [2016-02-22T21:05:09.209714 #30561]  INFO -- Kitchen: -----> Converging <monitoring-adaptly-ami-26>...
I, [2016-02-22T21:05:09.223345 #30561]  INFO -- Kitchen: -----> Converging <swap-disabled-adaptly-ami-26>...
I, [2016-02-22T21:05:10.422679 #30561]  INFO -- Kitchen: -----> Converging <ebs-reused-adaptly-ami-26>...
I, [2016-02-22T21:05:11.092862 #30561]  INFO -- Kitchen: -----> Converging <oom-adaptly-ami-26>...
I, [2016-02-22T21:05:11.472830 #30561]  INFO -- Kitchen: -----> Converging <monit-and-forwarding-adaptly-ami-26>...
I, [2016-02-22T21:05:14.476374 #30561]  INFO -- Kitchen: -----> Converging <ebs-adaptly-ami-26>...
I, [2016-02-22T21:05:34.099361 #30561]  INFO -- Kitchen: -----> Converging <swap-unavailable-adaptly-ami-26>...
I, [2016-02-22T21:05:34.146843 #30561]  INFO -- Kitchen: -----> Converging <swap-available-adaptly-ami-26>...
I, [2016-02-22T21:09:50.453548 #30561]  INFO -- Kitchen: -----> Setting up <oom-adaptly-ami-26>...
I, [2016-02-22T21:09:50.455006 #30561]  INFO -- Kitchen: -----> Verifying <oom-adaptly-ami-26>...
I, [2016-02-22T21:09:52.565687 #30561]  INFO -- Kitchen: -----> Setting up <monit-and-forwarding-adaptly-ami-26>...
I, [2016-02-22T21:09:52.567345 #30561]  INFO -- Kitchen: -----> Verifying <monit-and-forwarding-adaptly-ami-26>...
I, [2016-02-22T21:10:01.416699 #30561]  INFO -- Kitchen: -----> Setting up <swap-disabled-adaptly-ami-26>...
I, [2016-02-22T21:10:01.419707 #30561]  INFO -- Kitchen: -----> Verifying <swap-disabled-adaptly-ami-26>...
I, [2016-02-22T21:10:04.143476 #30561]  INFO -- Kitchen: -----> Setting up <swap-unavailable-adaptly-ami-26>...
I, [2016-02-22T21:10:04.145055 #30561]  INFO -- Kitchen: -----> Verifying <swap-unavailable-adaptly-ami-26>...
I, [2016-02-22T21:10:06.011496 #30561]  INFO -- Kitchen: -----> Destroying <oom-adaptly-ami-26>...
I, [2016-02-22T21:10:08.900980 #30561]  INFO -- Kitchen: -----> Setting up <ebs-reused-adaptly-ami-26>...
I, [2016-02-22T21:10:08.902438 #30561]  INFO -- Kitchen: -----> Verifying <ebs-reused-adaptly-ami-26>...
I, [2016-02-22T21:10:09.254307 #30561]  INFO -- Kitchen: -----> Destroying <monit-and-forwarding-adaptly-ami-26>...
E, [2016-02-22T21:10:09.756978 #30561] ERROR -- Kitchen: ------Exception-------
E, [2016-02-22T21:10:09.757023 #30561] ERROR -- Kitchen: Class: Kitchen::ActionFailed
E, [2016-02-22T21:10:09.757051 #30561] ERROR -- Kitchen: Message: Failed to complete #create action: [undefined method `search' for #<JMESPath::Runtime:0x007fae1c033870>]
E, [2016-02-22T21:10:09.757077 #30561] ERROR -- Kitchen: ---Nested Exception---
E, [2016-02-22T21:10:09.757102 #30561] ERROR -- Kitchen: Class: NoMethodError
E, [2016-02-22T21:10:09.757124 #30561] ERROR -- Kitchen: Message: undefined method `search' for #<JMESPath::Runtime:0x007fae1c033870>
E, [2016-02-22T21:10:09.757147 #30561] ERROR -- Kitchen: ------Backtrace-------
E, [2016-02-22T21:10:09.757170 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/jmespath-1.1.3/lib/jmespath.rb:33:in `search'
E, [2016-02-22T21:10:09.757194 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/aws-sdk-resources-2.1.36/lib/aws-sdk-resources/operations.rb:67:in `extract'
E, [2016-02-22T21:10:09.757219 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/aws-sdk-resources-2.1.36/lib/aws-sdk-resources/operations.rb:61:in `call'
E, [2016-02-22T21:10:09.757235 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/aws-sdk-resources-2.1.36/lib/aws-sdk-resources/resource.rb:147:in `load'
E, [2016-02-22T21:10:09.757256 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/aws-sdk-resources-2.1.36/lib/aws-sdk-resources/resource.rb:120:in `data'
E, [2016-02-22T21:10:09.757279 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/aws-sdk-resources-2.1.36/lib/aws-sdk-resources/resource.rb:223:in `block in add_data_attribute'
E, [2016-02-22T21:10:09.757302 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/kitchen-ec2-0.10.0/lib/kitchen/driver/aws/instance_generator.rb:131:in `debug_if_root_device'
E, [2016-02-22T21:10:09.757333 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/kitchen-ec2-0.10.0/lib/kitchen/driver/aws/instance_generator.rb:119:in `block_device_mappings'
E, [2016-02-22T21:10:09.757348 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/kitchen-ec2-0.10.0/lib/kitchen/driver/aws/instance_generator.rb:54:in `ec2_instance_data'
E, [2016-02-22T21:10:09.757371 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/kitchen-ec2-0.10.0/lib/kitchen/driver/ec2.rb:288:in `submit_server'
E, [2016-02-22T21:10:09.757391 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/kitchen-ec2-0.10.0/lib/kitchen/driver/ec2.rb:192:in `create'
E, [2016-02-22T21:10:09.757411 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/test-kitchen-1.4.2/lib/kitchen/instance.rb:424:in `public_send'
E, [2016-02-22T21:10:09.757423 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/test-kitchen-1.4.2/lib/kitchen/instance.rb:424:in `block in perform_action'
E, [2016-02-22T21:10:09.757433 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/test-kitchen-1.4.2/lib/kitchen/instance.rb:488:in `call'
E, [2016-02-22T21:10:09.757443 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/test-kitchen-1.4.2/lib/kitchen/instance.rb:488:in `synchronize_or_call'
E, [2016-02-22T21:10:09.757454 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/test-kitchen-1.4.2/lib/kitchen/instance.rb:453:in `block in action'
E, [2016-02-22T21:10:09.757464 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/2.2.0/benchmark.rb:288:in `measure'
E, [2016-02-22T21:10:09.757474 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/test-kitchen-1.4.2/lib/kitchen/instance.rb:452:in `action'
E, [2016-02-22T21:10:09.757484 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/test-kitchen-1.4.2/lib/kitchen/instance.rb:424:in `perform_action'
E, [2016-02-22T21:10:09.757494 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/test-kitchen-1.4.2/lib/kitchen/instance.rb:352:in `create_action'
E, [2016-02-22T21:10:09.757504 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/test-kitchen-1.4.2/lib/kitchen/instance.rb:341:in `block in transition_to'
E, [2016-02-22T21:10:09.757514 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/test-kitchen-1.4.2/lib/kitchen/instance.rb:340:in `each'
E, [2016-02-22T21:10:09.757525 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/test-kitchen-1.4.2/lib/kitchen/instance.rb:340:in `transition_to'
E, [2016-02-22T21:10:09.757534 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/test-kitchen-1.4.2/lib/kitchen/instance.rb:160:in `verify'
E, [2016-02-22T21:10:09.757545 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/test-kitchen-1.4.2/lib/kitchen/instance.rb:189:in `block in test'
E, [2016-02-22T21:10:09.757555 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/2.2.0/benchmark.rb:288:in `measure'
E, [2016-02-22T21:10:09.757565 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/test-kitchen-1.4.2/lib/kitchen/instance.rb:185:in `test'
E, [2016-02-22T21:10:09.757575 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/test-kitchen-1.4.2/lib/kitchen/command.rb:176:in `public_send'
E, [2016-02-22T21:10:09.757601 #30561] ERROR -- Kitchen: /home/ubuntu/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/test-kitchen-1.4.2/lib/kitchen/command.rb:176:in `block (2 levels) in run_action'
E, [2016-02-22T21:10:09.757612 #30561] ERROR -- Kitchen: ----------------------
```

**Comment by cheeseplus** [comment]
This appears to be an upstream bug https://github.com/aws/aws-sdk-ruby/issues/833 with AWS SDK Ruby so not much that we can do by my reading of the issue.

**Comment by hfinucane** [comment]
aws-sdk claims they've [shipped the fix](https://github.com/aws/aws-sdk-ruby/commit/79f8170883e71ae5183d882c2dbe1a158402cf98), but I can still reproduce with v2.3.7 of aws-sdk and test kitchen 1.4.2 & 1.9.0.

**Comment by hfinucane** [comment]
Does contributing to kitchen-ec2 require copyright assignment?

**Comment by cheeseplus** [comment]
It shouldn't as the Apache license already covers that - only copyleft (GPLish) licenses require assignment by my understanding.

**Comment by hfinucane** [comment]
Chef proper is Apache licensed, and requires copyright assignment- it's about the org, not the license.

**Comment by cheeseplus** [comment]
Chef doesn't own test-kitchen, it's an independent project and org.

**Comment by coderanger** [comment]
Also for the record, Chef has never required copyright assignment.

**Comment by cheeseplus** [comment]
Closed by #255
